eth-bytecode-db/eth-bytecode-db-server/src/settings.rs

```diff
@@ -48,7 +48,15 @@ pub struct DatabaseSettings {
 #[serde(deny_unknown_fields)]
 pub struct VerifierSettings {
     #[serde_as(as = "DisplayFromStr")]
-    pub uri: tonic::transport::Uri,
+    pub http_url: url::Url,
+    #[serde(default = "default_verifier_max_retries")]
+    pub max_retries: u32,
+    #[serde(default)]
+    pub validate_url: bool,
```

Comment:
validate url means that you check that provided url is indeed a URL. But you are sending health request, so I suggest to call it`check_health`

Reply:
Updated to `probe_url`